.gitmodules

```diff
@@ -78,4 +78,11 @@
 [submodule "src/BLAKE3"]
 	path = src/BLAKE3
 	url = https://github.com/BLAKE3-team/BLAKE3.git
-
+[submodule "src/boost_redis"]
+	path = src/boost_redis
+	url = https://github.com/boostorg/redis.git
+[submodule "src/nvmeof/gateway"]
+	path = src/nvmeof/gateway
```

Comment:
this repository has a spdk submodule that duplicates ceph's spdk submodule. maybe we can remove the original one?

Reply:
> maybe we can remove the original one?

+1.  I could be wrong, but I think it was added for some experimental work in Bluestore that never panned out.  We obviously need the SPDK fork in the organization, but we shouldn't need the submodule in ceph.git.